cpp/src/parquet/column_reader_test.cc

```diff
@@ -303,28 +303,40 @@ TEST_F(TestPrimitiveReader, TestSkipAroundPageBoundries) {
       values_.begin() + 4 * levels_per_page);
   ASSERT_TRUE(vector_equal(sub_values, vresult));
 
+  // 3) skip_size == page_size (skip page 4 from start of the page to the end)
+  levels_skipped = reader->Skip(levels_per_page);
+  ASSERT_EQ(levels_per_page, levels_skipped);
+  // Read half a page (page 5 to 5.5)
+  reader->ReadBatch(levels_per_page / 2, dresult.data(), rresult.data(), vresult.data(),
+                    &values_read);
+  sub_values.clear();
+  sub_values.insert(
+      sub_values.end(),
+      values_.begin() + static_cast<int>(5 * static_cast<double>(levels_per_page)),
+      values_.begin() + 5.5 * levels_per_page);
+  ASSERT_TRUE(vector_equal(sub_values, vresult));
+
   // 3) skip_size < page_size (skip limited to a single page)
```

Comment:
Can you fix the numbering in the comments? :-)

Reply:
Done.